lld/test/COFF/precomp-summary-fail.test

```diff
@@ -0,0 +1,22 @@
+
+The input files were tailored so that we end up with a resulting IPI stream
+smaller than the TPI stream, which would previously trigger a crash with
+/summary.
+
+RUN: rm -rf %t && mkdir %t
+RUN: yaml2obj < %S/Inputs/precomp2.yaml -o %t\precomp2.obj
+RUN: yaml2obj < %S/Inputs/precomp2-a.yaml -o %t\precomp2-a.obj
+RUN: lld-link %t\precomp2-a.obj %t\precomp2.obj /nodefaultlib /noentry \
+RUN:    /dll /out:%t.dll /debug /summary | FileCheck %s -check-prefix SUMMARY
+
+SUMMARY:                                     Summary
+SUMMARY-NEXT: --------------------------------------------------------------------------------
+SUMMARY-NEXT:               2 Input OBJ files (expanded from all cmd-line inputs)
+SUMMARY-NEXT:               0 PDB type server dependencies
+SUMMARY-NEXT:               1 Precomp OBJ dependencies
+SUMMARY-NEXT:               3 Merged TPI records
```

Comment:
@tstellar It looks like you've applied the original patch at https://reviews.llvm.org/rG4140f0744fb2deccb74e77282e23ff731f67821b which doesn't work on release/11.x. Please use the patch attached to https://bugs.llvm.org/show_bug.cgi?id=47754 instead! Let me know if it still doesn't work.